### PR TITLE
Testing: Sumator Tests

### PR DESCRIPTION
**Pull request by denisdimitrov**

### CODE REVIEW

File: test/SumatorTests.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        Sumator sum = new Sumator("-5").plus(new Sumator("-2"));
+        assertEquals("-7", sum.stringValue());
+    }
+
+    @Test
+    public void testWrongSymbol() throws Exception {
+        boolean thrown = false;
+        try {
+            new Sumator("ASD").plus(new Sumator("123"));
+        } catch (NumberFormatException z) {
```
Comment (EmilGeorgiev) [inline test/SumatorTests.java]: using try catch in test is bad. Use ```@Test(expected = NumberFormatException.class)``` to expect that this exception is throw

File: test/SumatorTests.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        }
+        assertTrue(thrown);
+    }
+
+    @Test
+    public void testNullArgument() throws Exception {
+        boolean thrown = false;
+        try {
+            new Sumator(null).plus(new Sumator(null));
+        } catch (IllegalArgumentException z) {
```
Comment (EmilGeorgiev) [inline test/SumatorTests.java]: .

**Comment by denisdimitrov** [comment]
I removed the try/catch blocks.

**Comment by EmilGeorgiev** [comment]
You should add documentation of methods and classes.

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request merged**